modules/ROOT/pages/acm-lightning-components.adoc

```diff
@@ -68,37 +68,34 @@ Filter Condition (Advanced):: Causes the carousel to show only APIs that match t
 
-* The API name is exactly equal to `My API`: `+acm_pkg__Name__c = 'My API'+`
-* The API name contains the text `My API`: `+acm_pkg__Name__c LIKE '%My API%'+`
-* The API name is `API Alpha` or `API Bravo`: `+(acm_pkg__Name__c = 'API Alpha' OR acm_pkg__Name__c = 'API Bravo')+`
-* The API name is `My API` and the API version group is `v1`: `+(acm_pkg__Name__c = 'My API' AND acm_pkg__VersionGroup__c = 'v1')+`
-* The API name begins with `Product`, `Shipping`, or `Order`: `+(acm_pkg__Name__c LIKE 'Product%' OR acm_pkg__Name__c LIKE 'Shipping%' OR acm_pkg__Name__c LIKE 'Order%')+`
-* The API has an icon: `+acm_pkg__Icon__c != ''+`
-* The API has a Learn More button that links to a custom page with more information: `+acm_pkg__NavigateUrl__c != ''+`
-* The API version group is exactly equal to `v2`: `+acm_pkg__VersionGroup__c = 'v2'+`
-* The API version group is later than `v1`: `+acm_pkg__VersionGroup__c > 'v1'+`
-* The API version group is earlier than `v1`: `+acm_pkg__VersionGroup__c < 'v1'+`
-* The API version group is later than or equal to `v2`: `+acm_pkg__VersionGroup__c >= 'v2'+`
-* The API version group is earlier than or equal to `v2`: `+acm_pkg__VersionGroup__c <= 'v2'+`
+A card may have multiple badges from one category. A card displays up to three badges, and if there are more, offers a tooltip to display them.
 
-Queries using `OR` or `AND` must enclose the whole condition in parentheses.
+If category colors are not specified, the community theme is used for defaults. Colors can be hexadecimal values such as `#FF0000` or `#0000FF`, or standard CSS color names such as `red` and `blue`.
 
-ACM compares version groups using normal string comparison.
+URL parameters can be used to pass filters to the catalog. For example, the URL of the page with the catalog component can end with `?catalogFilters=Country:Argentina,USA;Payment:Cash,Credit`.
 
-You can learn more about SOQL from the Salesforce https://developer.salesforce.com/docs/atlas.en-us.soql_sosl.meta/soql_sosl/[SOQL and SOSL Reference, role=external, window=blank].
+The API icon, name, and description shown are set in the ACM control panel when you xref:publish-apis.adoc[add an API version to your community].
+
+You can specify values for the following parameters of the API Carousel:
+
+Component Title:: Specifies the catalog title. Default is *API Catalog*.
+Show API Counter:: Enables you to show the API counter, which shows the number of APIs that match the current search and filters.
+APIs Per Page:: Specifies the number of APIs shown on each page of the catalog. Default is 12.
+Category Filter 1 through Category Filter 5:: Specifies the categories community members and guest users can use to filter the APIs. Default is *None*.
+Category Color 1 through Category Color 5:: Specifies the color for each category.
+Search Input Position:: Specifies whether the search box will be next to the filters or below the filters and above the cards.
```

Comment:
```suggestion
Search Input Position:: Specifies whether the search box is next to the filters or below the filters and above the cards.
```

Reply:
Argh, that was a foolish mistake. Thank you!